### PR TITLE
[All hosts](Update sample links)

### PR DESCRIPTION
**Pull request by davidchesnut**
Changes from the old PnP-OfficeAddins URL to Office-Add-in-samples URL.

### CODE REVIEW

File: docs/develop/dialog-api-in-office-add-ins.md [hunk 1]
```diff
@@ -435,18 +435,18 @@ All of the following samples use `displayDialogAsync`. Some have NodeJS-based se
...
-- [Outlook Add-in SSO](https://github.com/OfficeDev/PnP-OfficeAddins/tree/main/Samples/auth/Outlook-Add-in-SSO)
+- [Outlook Add-in Microsoft Graph ASPNET](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Outlook-Add-in-Microsoft-Graph-ASPNET)
+- [Outlook Add-in SSO](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Outlook-Add-in-SSO)
 - [Outlook Add-in Token Viewer](https://github.com/OfficeDev/Outlook-Add-In-Token-Viewer)
 - [Outlook Add-in Actionable Message](https://github.com/OfficeDev/Outlook-Add-In-Actionable-Message)
 - [Outlook Add-in Sharing to OneDrive](https://github.com/OfficeDev/Outlook-Add-in-Sharing-to-OneDrive)
 - [PowerPoint Add-in Microsoft Graph ASPNET InsertChart](https://github.com/OfficeDev/PowerPoint-Add-in-Microsoft-Graph-ASPNET-InsertChart)
-- [Excel Shared Runtime Scenario](https://github.com/OfficeDev/PnP-OfficeAddins/tree/900b5769bca9bbcff79d6cd6106d9fcc55c70d5a/Samples/excel-shared-runtime-scenario)
+- [Excel Shared Runtime Scenario](https://github.com/OfficeDev/Office-Add-in-samples/tree/900b5769bca9bbcff79d6cd6106d9fcc55c70d5a/Samples/excel-shared-runtime-scenario)
```
Comment (AlexJerabek) [inline docs/develop/dialog-api-in-office-add-ins.md:449]: Why does this one not link to the `main` branch?

File: docs/develop/support-ie-11.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ There are two popular transpilers. Both of them can work with source files that
...
+See the documentation for either of them for information about installing and configuring the transpiler in your add-in project. We recommend that you use a task runner, such as [Grunt](https://gruntjs.com/) or [WebPack](https://webpack.js.org/) to automate the transpilation. For a sample add-in that uses tsc, see [Office Add-in Microsoft Graph React](https://github.com/OfficeDev/Office-Add-in-samples/tree/3ce0e1b74152dbbe8306a091696bc4455c04c0a1/Samples/auth/Office-Add-in-Microsoft-Graph-React). For a sample that uses babel, see [Offline Storage Add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/3ce0e1b74152dbbe8306a091696bc4455c04c0a1/Samples/Excel.OfflineStorageAddin).
```
Comment (AlexJerabek) [inline docs/develop/support-ie-11.md:38]: Why does this one not link to the main branch?

File: docs/overview/office-add-in-code-samples.md [hunk 3]
```diff
@@ -13,55 +13,55 @@ These code samples are written to help you learn how to use various features whe
...
 
 ## Outlook
 
 | Name                | Description         |
 |:--------------------|:--------------------|
-| [Use Outlook event-based activation to tag external recipients (preview)](/samples/officedev/pnp-officeaddins/outlook-add-in-tag-external-recipients) | Use event-based activation to run an Outlook add-in when the user changes recipients while composing a message. The add-in also uses the `appendOnSendAsync` API to add a disclaimer. |
-| [Use Outlook event-based activation to set the signature](/samples/officedev/pnp-officeaddins/outlook-add-in-set-signature/) | Use event-based activation to run an Outlook add-in when the user creates a new message or appointment. The add-in can respond to events, even when the task pane is not open. It also uses the `setSignatureAsync` API. |
+| [Use Outlook event-based activation to tag external recipients (preview)](/samples/officedev/Office-Add-in-samples/outlook-add-in-tag-external-recipients) | Use event-based activation to run an Outlook add-in when the user changes recipients while composing a message. The add-in also uses the `appendOnSendAsync` API to add a disclaimer. |
```
Comment (AlexJerabek) [inline docs/overview/office-add-in-code-samples.md:25]: What's the motivation for linking to the samples page instead of the GitHub repo in some of these?

**Review by AlexJerabek — APPROVED** [review]
Approving with a couple questions.

File: docs/overview/office-add-in-code-samples.md (hunk 3 above)
Comment (davidchesnut) [reply docs/overview/office-add-in-code-samples.md:25]: Probably will change these in a future PR. I'm running an experiment on this.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**